.travis.yml

```diff
@@ -68,3 +72,25 @@ before_cache:
 notifications:
   slack:
     secure: V5eyoGShxFoCcYJcp858vf/T6gC9KeMxL0C1EElcpZRcKBrIVZzvhek3HLHxZOxlghqnvNVsyDtU3u5orkEaAXeXj5c2dN+4XBsAB9oeN5MtQ0Z3VLAhZDqKIW1LzcXrq4DpzM0PkGhjfjum/P94/qFYk0UckPtB6a341AuYRo8=
+
+before_deploy:
+  - mkdir /tmp/artifacts
+  - cp presto-server/target/presto-server-*.tar.gz /tmp/artifacts
+  - cp presto-server-rpm/target/presto-server-rpm-*.x86_64.rpm /tmp/artifacts
+  - cp presto-product-tests/target/presto-product-tests-*-executable.jar /tmp/artifacts
+  - cp presto-jdbc/target/presto-jdbc-*.jar /tmp/artifacts
+  - cp presto-cli/target/presto-cli-*-executable.jar /tmp/artifacts
+  - ls -lah /tmp/artifacts
+
+deploy:
+  on:
+    all_branches: true
+    condition: -v DEPLOY_S3_SECRET_KEY_ID && -v PRODUCT_TESTS
+  provider: s3
+  access_key_id: ${DEPLOY_S3_SECRET_KEY_ID}
+  secret_access_key: ${DEPLOY_S3_SECRET_KEY}
+  bucket: ${DEPLOY_S3_BUCKET}
+  skip_cleanup: true
+  local-dir: /tmp/artifacts
+  upload-dir: travis_build_artifacts/${TRAVIS_REPO_SLUG}/${TRAVIS_BUILD_NUMBER}/${TRAVIS_JOB_NUMBER}
```

Comment:
I would prefer to have following path:
`travis_build_artifact/<org>/<project>/<branch/<job_number>